src/ec/packages/ec-component-page-banner/page-banner.story.js

```diff
@@ -1,65 +1,161 @@
 import { storiesOf } from '@storybook/html';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
-import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
+import {
+  withKnobs,
+  text,
+  boolean,
+  select,
+  button,
+} from '@storybook/addon-knobs';
 import withCode from '@ecl-twig/storybook-addon-code';
-import { getExtraKnobs, tabLabels, getIconKnobs } from '@ecl-twig/story-utils';
+import {
+  getExtraKnobs,
+  tabLabels,
+  getIconKnobs,
+  getComplianceKnob,
+} from '@ecl-twig/story-utils';
 
 import uiIcons from '@ecl/ec-resources-icons/dist/lists/ui.json';
-
 import bannerDataDefault from './demo/data--default';
 import bannerDataImage from './demo/data--image';
 import bannerDataImageShade from './demo/data--image-shade';
 import bannerDataPrimary from './demo/data--primary';
 import bannerDataAlignLeft from './demo/data--align-left';
-
 import pageBanner from './ecl-page-banner.html.twig';
 import notes from './README.md';
 
+// Preserve the adapted specs.
+let defaultData = { ...bannerDataDefault };
+let imageData = { ...bannerDataImage };
+let imageShadeData = { ...bannerDataImageShade };
+let primaryData = { ...bannerDataPrimary };
+let leftData = { ...bannerDataAlignLeft };
+
+// Show/hide buttons for optional elements.
+const defaultDescBtnToggler = () => {
+  defaultData.baseline = defaultData.baseline
```

Comment:
It's the first time I see boolean false on the place of ternary true expression statement. Hard to read and follow.
Please find a way to rewrite to have the conventional `condition ? exprIfTrue : exprIfFalse` or `||` as `defaultData.baseline = bannerDataDefault.baseline || defaultData.baseline;`

Reply:
I'm still trying to understand the logic here, really hard.
Do you try to return false because of button handlers? Reading docs of the addon
```
Button knobs cause the story to re-render after the handler fires. You can prevent this by having the handler return false.
```

Do you try to set false for a property which is managed by a boolean knob otherwise?

Reply:
There might be a better way for doing this, but this are toggler buttons, the way they act is to remove a piece of data, the false it's set to hide the "element" when the data is in place, while the copied data from the cloned object is used to restore it when the user clicks again.
I refactored it with ifs to avoid confusion, deleting the piece of data instead of setting in to false.

Buttons in twig are used in the "use cases" tab, to quickly show the different look and feel of a component when it's relevant, normally we use them for the big ones, and this one seemed relevant due to the fact that there is nothing mandatory, basically. 
It's true that they overlap the work by some knob and they don't play well together, though.